db/schema/patch-008-007.sql

```diff
@@ -0,0 +1,18 @@
+-- -- drop new stored procedures
+
+-- DROP PROCEDURE `lockAccount_1`;
+-- DROP PROCEDURE `unlockAccount_1`;
+-- DROP PROCEDURE `unlockCode_1`;
+-- DROP PROCEDURE `emailRecord_2`;
+-- DROP PROCEDURE `account_2`;
+-- DROP PROCEDURE `forgotPasswordVerified_3`;
+-- DROP PROCEDURE `deleteAccount_3`;
+-- DROP PROCEDURE `resetAccount_3`;
+
+-- -- drop new column
+-- ALTER TABLE accounts DROP COLUMN lockedAt;
+
+-- -- drop new table
+-- DROP TABLE accountUnlockCodes
+
```

Comment:
General note: I try and make sure that every statement in the reverse patch is in exactly the reverse order of the original patch. This makes sure we 'unwind' exactly the same every time.

In this case the order of the `DROP PROCEDURES` it totally doesn't matter here, nor does the `ALTER TABLE` or `CREATE/DROP TABLE` but just in case there were any weird interactions between anything it seems safer to do so.